### PR TITLE
Landing page text 

### PR DESCRIPTION
**Pull request by adityaekawade**

### CODE REVIEW

**Comment by adityaekawade** [comment]
Test url: http://dev.clin.iobio.io/

**Comment by mvelinder** [comment]
Let's lower case the C in Review case and the V in Review variants

image

to match the landing page

**Comment by adityaekawade** [comment]
Fixed 
Screen Shot 2020-03-30 at 4 15 05 PM

**Comment by tonydisera** [comment]
Adit, it looks like the labels on the landing page are still out of sync.  See comments in issue #132

**Comment by tonydisera** [comment]
Yes Adit.  Please see the screen print provided in the issue.  I test everything locally when I review a pull request.

**Comment by tonydisera** [comment]
git clone https://github.com/iobio/clin.iobio.git -b update-landing-page-text

>npm install
>npm start

Screen Shot 2020-03-30 at 8 12 06 PM

File: src/components/pages/LandingPage.vue [hunk 1]
```diff
@@ -182,35 +182,35 @@ export default {
       carouselData: 0,
       step_number: 0,
       workflow_steps: [
-        { text: 'Check case' },
+        { text: 'Review case' },
         { text: 'Select phenotypes' },
-        { text: 'Investigate variants' },
-        { text: 'Review findings' },
+        { text: 'Review variants' },
+        { text: 'Report findings' },
```
Comment (adityaekawade) [inline src/components/pages/LandingPage.vue]: Hi @tonydisera , 
Can you test once again to see if the wordings are synced? I cloned the repo to verify and seems to be showing the updated text. Also, you can see here that the text is changed. 

Screen Shot 2020-03-31 at 10 34 57 AM

**Pull request merged**